src/Tasks.UnitTests/RoslynCodeTaskFactory_Tests.cs

```diff
@@ -644,6 +644,124 @@ public override bool Execute()
             }
         }
 
+        [Fact]
+        public void EmbedsSourceFileInBinlog()
+        {
+            string taskName = "HelloTask";
+            string csprojFileName = "EmbedsSourceFileInTheBinlogTest.csproj";
+            string targetName = "SayHello";
+
+            var env = TestEnvironment.Create();
```

Comment:
This is an `IDisposable` that cleans up on dispose

```suggestion
            using var env = TestEnvironment.Create();
```